src/inverseRobot.js

```diff
@@ -8,6 +8,16 @@
 
 function inverseRobot(robot) {
   // write code here
-}
+  const inverse = {};
+
+  for (const key in robot) {
+    if (Object.prototype.hasOwnProperty.call(inverse, robot[key])) {
+      return null;
+    }
 
+    inverse[robot[key]] = key;
```

Comment:
Consider adding a comment here to explain that you're inverting the keys and values of the robot object.